gui/src/routes/_home.home/transactions.lazy.tsx

```diff
@@ -223,24 +219,40 @@ function Details({ tx, chainId }: DetailsProps) {
         <>
           <Datapoint
             label="maxFeePerGas"
-            value={`${fullTx.maxFeePerGas && formatGwei(BigInt(fullTx.maxFeePerGas))} gwei`}
+            value={
+              fullTx.maxFeePerGas ? (
+                <NumberView value={Number(fullTx.maxFeePerGas)} />
```

Comment:
this seems incorrect. `maxFeePerGas` and `maxPriorityFeePerGas` are already expressed in `gwei`, but that information is lost here

I'd expect something along the lines of

`<NumberView value={fullTx.maxFeePerGas} unit="gwei" />`

where the component would be smart enough to show me the conversions from there